server/common/rest.py

```diff
@@ -14,6 +14,13 @@
 from server.data_common.fbs.matrix import decode_matrix_fbs
 
 
+def abort_and_log(code, logmsg, loglevel=logging.DEBUG):
+    """ Debug-level log the message, then abort with HTTP code """
+    current_app.logger.log(loglevel, logmsg)
```

Comment:
One improvement here would be to get the exception into the log file.   We could have a variant of this function that captures the exception information.  Maybe for a later PR.

Reply:
very easy to do - I'll add it now.